server/keyshare/tasks/tasks.go

```diff
@@ -128,16 +128,20 @@ func (t *taskHandler) sendExpiryEmails(ctx context.Context, id int64, username,
 			}
 
 			if err := keyshare.VerifyMXRecord(email); err != nil {
-				if t.revalidateMail {
-					if err = t.db.ExecUserContext(ctx, "UPDATE irma.emails SET revalidate_on = $1 WHERE id = $2",
-						time.Now().AddDate(0, 0, 5).Unix(),
-						id); err != nil {
-						t.conf.Logger.WithField("error", err).Error("Could not update email address to set revalidate_on")
-						return err
-					}
+				if !t.revalidateMail {
+					// It could be this user has another, valid email address. Therefore we do not abort
+					// since is a chance we can still inform the user.
+					return nil
```

Comment:
Why was this change necessary? If you return the error, then [this check](https://github.com/privacybydesign/irmago/pull/344/files#diff-d2930c933338a39b361fd236af2fe82f5741b7a689ef97a330534c1bb8cd2b58R229) ensures that the user is skipped, as we want.